recipes/geonum/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "geonum" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f393389a0c22664613075cd22e784aee264e7ea4c4d8aa2accf2b2896c826150
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Comment:
Please use the sting from the example: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

Reply:
Fixed!